editoast/editoast_derive/src/model/codegen/changeset_decl.rs

```diff
@@ -38,15 +38,22 @@ impl ToTokens for ChangesetDecl {
                 np!(column_name, vis, name, ty)
             })
             .unzip();
-        tokens.extend(quote! {
-            #[derive(Debug, Default, Queryable, AsChangeset, Insertable, #(#additional_derives),*)]
-            #[diesel(table_name = #table)]
-            #vis struct #ident {
-                #(
-                    #[diesel(deserialize_as = #field_type, column_name = #field_column)]
-                    #field_vis #field_name: Option<#field_type>
-                ),*
-            }
-        });
+        if fields.is_empty() {
```

Comment:
Maybe add a comment to explain why we do need this `if`?

Suggestion:

> If the struct has no fields, `Queryable`, `AsChangeset` and `Insertable` cannot be derived.

It should help future developers to quickly see where the difference is and why.

Reply:
d2c4bc841